src/chi/components/sidebar/sidenav.scss

```diff
@@ -376,6 +398,134 @@ $sidenav-lg-width: 12.5rem;
+                    background-color: rgba(set-color(mint, 40), 0.08);
+
+                    &.unselected {
+                      background-color: transparent;
+                      box-shadow: none;
+                    }
+                  }
+                }
+
+                &:not(.-active) {
+                  a.-active {
+                    background-color: set-color(grey, 90);
+                    box-shadow: inset 0.125rem 0 0 set-color(white);
+                  }
+                }
+              }
+            }
+          }
+        }
+      }
+
+      .m-sidenav__drawers {
+        .m-drawer {
+          background-color: set-color(grey, 90);
+
+          &.-active {
+            .m-drawer__header {
+              align-items: center;
+              background-color: rgba(set-color(grey, 100), 0.5);
+              color: set-color(grey, 40);
+              display: flex;
+              font-size: 16px;
+              height: 4rem;
+              padding-left: 1.5rem;
+
+              .a-drawer__title {
+                font-weight: 600;
+                left: 1.25rem;
+              }
+
+              .a-btn {
+                &.-close {
+                  right: 1rem;
+                  top: 1rem;
+                }
+              }
+            }
+
+            .m-drawer__content {
+              background-color: set-color(grey, 90);
+
+              .m-sidenav__drawer-list {
+                border-top: 0;
+                padding-top: 0.5rem;
+
+                li {
+                  border-bottom: 0;
+                  line-height: 20px;
+
+                  a {
+                    color: set-color(white);
+                    padding-bottom: 0.625rem;
+                    padding-left: 1.5rem;
+                    padding-top: 0.625rem;
+
+                    .m-sidenav__title {
+                      font-weight: 400;
+                    }
+                  }
+
+                  &.-active {
+                    & > a {
+                      background-color: set-color(grey, 80);
+                      box-shadow: none;
+
+                      .m-sidenav__title {
+                        font-weight: 600;
+                      }
+                    }
+
+                    .m-sidenav__drawer-item-list {
+                      display: block;
+                    }
+                  }
+
+                  .m-sidenav__drawer-item-list {
+                    background-color: set-color(grey, 80);
+
+                    .a-tabs {
+                      &.-sm {
+                        li {
+                          a {
+                            &.a-sidenav__drawer-item-tab {
+                              font-size: 13px;
+                              font-weight: 400;
+                              line-height: 20px;
+                              padding-bottom: 0.375rem;
+                              padding-left: 3rem;
+                              padding-top: 0.375rem;
+                            }
```

Comment:
In their design a hover state is defined on `m-sidenav__drawer-item-list` tab items. Let's add:
```
&:hover {
   background-color: set-color(grey, 70);
}
```